packages/@okta/vuepress-site/code/library-versions/index.md

```diff
@@ -81,7 +81,6 @@ This table lists the current (and historical) major versions of our open-source
 
 [sdk-angular]: https://github.com/okta/okta-oidc-js/tree/master/packages/okta-angular
 [sdk-dotnet]: https://github.com/okta/okta-sdk-dotnet
-[sdk-dotnet-033]: https://github.com/okta/okta-sdk-dotnet/releases/tag/0.3.3.0
```

Comment:
Should there be a link to the currently tagged version?

Reply:
I think this is only for this particular version that is retired. Other libraries don't seem to be using a link to the current tagged version.